tests/telemetry/conftest.py

```diff
@@ -17,6 +18,22 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope="module", autouse=True)
+def setup_user_auth(duthosts, enum_rand_one_per_hwsku_hostname):
+    """
+    Setup user authentication for telemetry tests
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    env = GNMIEnvironment(duthost, GNMIEnvironment.TELEMETRY_MODE)
```

Comment:
What is the difference between setting client_auth to false and setting user_auth to none? Same ending behavior?